tajo-catalog/tajo-catalog-common/src/main/java/org/apache/tajo/catalog/SchemaUtil.java

```diff
@@ -161,4 +161,52 @@ public static String toDisplayString(Schema schema) {
     DDLBuilder.buildSchema(sb, schema);
     return sb.toString();
   }
+
+  public static int estimateSchemaSize(Schema schema) {
```

Comment:
It seems to need some comment because it is hard to image its return value from the function name.
Also, ```estimateRowSize``` rather than ```schema``` may be more intuitive.